app/models/account.rb

```diff
@@ -75,7 +75,7 @@ class Account < ApplicationRecord
   validates_with UniqueUsernameValidator, if: -> { local? && will_save_change_to_username? }
   validates_with UnreservedUsernameValidator, if: -> { local? && will_save_change_to_username? }
   validates :display_name, length: { maximum: 30 }, if: -> { local? && will_save_change_to_display_name? }
-  validates :note, note_length: { maximum: 160 }, if: -> { local? && will_save_change_to_note? }
+  validates :note, note_length: { maximum: Setting.max_bio_chars.to_i }, if: -> { local? && will_save_change_to_note? }
```

Comment:
When the setting has never been set before in the database and it comes from the default yaml it will merge from the ruby side as a string and this just coerces it so the JavaScript is an explicit int just like it was when it was hard coded to 160.